landing-page.css

```diff
@@ -0,0 +1,90 @@
+
+.main {
+	padding: 1rem 3rem;
+	grid-area: main;
+	width: 100%;
+	box-sizing: border-box;
+	max-width: 100vw;
+}
+
+.body-container {
+	margin-bottom: 1.5rem;
+}
+
+.grid-container-pagelayout {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+	grid-template-areas:
+		'header header header header header'
+		'sidebar main main main main'
+		'sidebar main main main main'
+		'sidebar main main main main';
+	margin-bottom: 1.5rem;
+}
+
+hr {
+	margin: 3rem 0 2rem;
+}
+
+h2 {
+	padding-top: 1rem;
+}
+.spacer-2rem {
+	height: 2rem;
+	width: 100%;
+}
+
+footer {
+	background-color: var(--TERTIARY);
+	min-height: 7rem;
+	width: 100%;
+	padding: 0.625rem;
+	color: white;
+	font-size: 1.5rem;
+	box-sizing: border-box;
+}
+.footer-social-icons {
+	width: fit-content;
+	margin: 1rem auto;
+}
+.footer-social-icons a {
+	transition: 0.2s ease;
+}
+.footer-social-icons a:hover {
+	color: var(--SECONDARY-S1);
+}
+
+.footer-header {
+	font-size: 1rem;
+	letter-spacing: 0.5px;
+}
+.footer-copy {
+	color: rgb(167, 167, 167);
+}
+.footer-copy strong, .logo-sm{
+    font-size: 1.5rem;
+}
+.logo-xsm {
+    font-size: 1rem;
```

Comment:
try to use logo-xs. It's understandable that it will be extra small. Always use 2characters for size like lg for large. xl for extra large

Reply:
done with the mentioned changes